.gitlab-ci.yml

```diff
@@ -4,11 +4,14 @@ before_script:
    - pip install tox
 
 stages:
-   - build
-
-tox:
-   stage: build
-   script:
-      - tox
-   tags:
-      - docker
+   - lint and unit test
+
+lint:
+  stage: lint and unit test
+  script:
+    - tox -e flake8
+
+unit tests:
+  stage: lint and unit test
+  script:
+    - tox -e py37
```

Comment:
Your pull request seem to contain changes which you haven't authored, and are already in master. I would suggest to inspect your changes and present the pull request so that your changes are rebased on top to current master branch.